### ISSUE TITLE
Where is the source code?

### ISSUE THREAD

**Issue by CesarPazzi** [ShekharSumanPro/NetSpeedMonitor#1]
Upload the source code to the repo

**Comment by ofarukcaki** [comment]
He is not the developer of this program and afaik it's not open source. I don't get the reason why he shared this program like that and I suspect of malicious software

**Comment by CesarPazzi** [comment]
yes I also suspect malware but Github does not do anything about, I report it since April but no action was taken.

**Comment by vicky-nike** [comment]
lol. his knowledge is his bank balance.
